### PR TITLE
chore: add advanced stale config for issues and labels

### PR DESCRIPTION
**Pull request by yamilmedina**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

After discussing in the collective, and because we want to have some fine-tuning for stale bot.
So we can be aware of issues and PRs that doesn't have updates to reduce the queue of prs and issues to review.

### Solutions

Inspired by **GitHub docs** action stale https://github.com/github/docs/issues/23181 we added a new action to mark issues and prs as stale, and only for Issues to be closed.

This also coniders draft PRs and WIP label

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: .github/workflows/stale-issues-and-pr.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    schedule:
+        -   cron: '10 0 * * *'
+
+jobs:
+    stale:
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/stale@v8
+                with:
+                    days-before-stale: 30
```
Comment (alexandreferris) [inline .github/workflows/stale-issues-and-pr.yml:17]: On the docs above it says 60 days but here is 30, is here wrong or are the docs above misleading? 🤔

File: .github/workflows/stale-issues-and-pr.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    days-before-close: 7
+
+                    stale-pr-message: 'This PR is stale because it has been open 30 days with no activity. Please update it or close it in case is not relevant anymore.'
```
Comment (alexandreferris) [inline .github/workflows/stale-issues-and-pr.yml:20]: here too

File: .github/workflows/stale-issues-and-pr.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                    stale-pr-label: 'stale'
+                    exempt-pr-labels: 'WIP'
+                    exempt-draft-pr: true
+                    days-before-pr-close: -1
+
+                    stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
```
Comment (alexandreferris) [inline .github/workflows/stale-issues-and-pr.yml:26]: 30 days here too

**Review by alexandreferris — COMMENTED** [review]
Nice! Just a quick question on the days is it 30 or 60? 🤔

File: .github/workflows/stale-issues-and-pr.yml (hunk 1 above)
Comment (yamilmedina) [reply .github/workflows/stale-issues-and-pr.yml:17]: fixed and added comments

**Review by alexandreferris — APPROVED** [review]
🚀

**Review by MohamadJaara — APPROVED** [review]

**Pull request merged**